sceptre/sageit/config/prod/ghasvcaccount.yaml

```diff
@@ -0,0 +1,8 @@
+template_path: remote/service-account.yaml
```

Comment:
If you want to create a service user with a custom policy then you'll need to use the cfn template in the not yet merged PR https://github.com/Sage-Bionetworks/aws-infra/pull/275

Reply:
Will refactor if I can actually remove the Jenkins system (staging is doable from gha but I'm not clear on prod deployment).